src/java.desktop/share/classes/javax/swing/SwingPaintEventDispatcher.java

```diff
@@ -32,25 +32,22 @@
 import sun.awt.AppContext;
 import sun.awt.SunToolkit;
 import sun.awt.event.IgnorePaintEvent;
-import sun.security.action.GetBooleanAction;
-import sun.security.action.GetPropertyAction;
```

Comment:
You can also remove the now-unused import of `AccessController` from this file as a follow-up.